src/components/ebay-textbox/template.marko

```diff
@@ -1,4 +1,81 @@
+<if(data.initIcons)>
+<span style="display: none">
```

Comment:
@rachelvillamin Thanks for the PR. Does including all the icon tags here cause a maintenance problem down the line? Meaning, developers should remember to update this list when adding a new icon.

Reply:
I had to declare the icons explicitly or the icons will not appear. I think this is a limitation on ebay-icon.

![screen shot 2018-11-15 at 11 06 06 am](https://user-images.githubusercontent.com/20077653/48520948-ac7f6980-e8c6-11e8-9e40-3b4efaec3d47.png)

Reply:
@DylanPiercey What do you think?

Reply:
Cool. Thanks Dylan! I'll look at ebay-menu and follow the same approach :)

Reply:
This is done. I'm now using a transformer. Thanks Dylan for our help especially with the body-if-only work-around.